src/vuex/account.module.js

```diff
@@ -33,6 +33,12 @@ export const actions = {
       include: ['balances.asset', 'balances.state'],
     })
 
+    commit(
+      vuexTypes.UPDATE_ASSETS,
+      account.balances.map(b => b.asset),
+      { root: true }
+    )
+
```

Comment:
Pls also add BalanceRecord. To keep backward compatibility, you can add a new getter for balances that will return you wrapped balances.

Reply:
Added